render/html/user.py

```diff
@@ -22,12 +22,18 @@ def login_disabled(self, authMethods, tpl=None, **kwargs):
 		template = self.getEnv().get_template(self.getTemplateFileName(tpl))
 		return template.render(authMethods=authMethods, **kwargs)
 
-	def login(self, authMethods, tpl=None,  **kwargs):
+	def login(self, skel, tpl=None, **kwargs):
 		if "loginTemplate" in dir(self.parent):
 			tpl = tpl or self.parent.loginTemplate
 		else:
 			tpl = tpl or self.loginTemplate
+		return self.add(skel, tpl=tpl)
 
+	def loginChoices(self, authMethods, tpl=None, **kwargs):
+		if "loginTemplate" in dir(self.parent):
```

Comment:
Is it correct to use the same template as for login here?

Reply:
That would be the "backward compatible" way. Decide inside the template. If we want to change this we'll have to decide which has to be renamed (my suggestion: loginChoicesTemplate)

Reply:
I think directly loginChoicesTemplate would be more straightforward.